File: C#/ExercicioRetanguloMathSqrtPow.cs
using System;

namespace atribuiçãoValores
{
    class Atribuição
    {
        static void Main(string[] args)
        {

            //DECLARAÇÃO VARIÁVEIS
            double largura, altura, area, perimetro, diagonal;

            //LEITURA INPUTS
            Console.Write("Base do retângulo: ");
            largura = double.Parse(Console.ReadLine());
            Console.Write("Altura do retângulo: ");
            altura = double.Parse(Console.ReadLine());

            //MANIPULAÇÃO DADOS
            area = largura * altura;
            perimetro = 2 * (largura + altura);
            diagonal = Math.Sqrt(Math.Pow(largura, 2.0) + Math.Pow(altura, 2.0));

            //PRINTANDO OUTPUTS
            Console.WriteLine("AREA = " + area.ToString("F4"));
            Console.WriteLine("PERIMETRO = " + perimetro.ToString("F4"));
            Console.WriteLine("DIAGONAL = " + diagonal.ToString("F4"));
        }
    }
}

File: C++/ExercicioRetanguloMathSqrtPow.cpp
// #include <bits/stdc++.h>
#include <iostream>
#include <cmath> //para sqrt e pow
#include <iomanip>

using namespace std;

int main()
{
    // DECLARAÇÃO VARIÁVEIS
    double area, perimeter, diagonal, base, height;

    // LEITURA INPUTS
    cout << ("Base do retângulo: ");
    cin >> base;
    cout << ("Altura do retângulo: ");
    cin >> height;

    // MANIPULAÇÃO DADOS
    area = (base * height);
    perimeter = 2 * (base + height);
    diagonal = sqrt((pow(base, 2.0)) + (pow(height, 2.0)));

    // PRINT OUTPUTS
    cout << fixed << setprecision(4);
    cout << "AREA = " << area << endl;
    cout << "PERIMETRO = " << perimeter << endl;
    cout << "DIAGONAL = " << diagonal << endl;

    return 0;
}